config/locales/views.en.yml

```diff
@@ -511,3 +579,13 @@ en:
       ban: "Ban Control"
       title: "Actions"
       list: "Manage list memberships"
+    profile:
+      badge:
+        admin: "Admin"
+        banned: "Banned"
+        follows_you: "Follows you"
+        moderator: "Moderator"
+  question:
+    show:
+      questionbox:
+        title: "This question was not in your inbox? Answer it here!"
```

Comment:
Final newline missing

Reply:
Also this string reads super non-English.

Reply:
```suggestion
        title: "Want to answer this ask?  Question yourself!  :E"
```